### DISCUSSION TITLE
Get positions and sizes from symbols

### DISCUSSION THREAD

**Discussion by Star-Frog** [lennart-finke/GodoTeX#15] [Q&A] [answered]
Hello, first of all thank you for publishing the godot module. I would like to ask if it is possible to get the positions and sizes of the variables and values in a simple way?

**Comment by lennart-finke** [comment, marked as answer]
Hello @Star-Frog, great to hear you're using the plug-in!

I understand your question like this: For a given rendered equation, say $x^2 \cdot y$, you would like a function to get bounding boxes of the pixels in the rendered Sprite2D which correspond to certain input tokens like "y" or "\cdot" or "2".

If I understood correctly, that is a good question, but the answer in the current version would have to be no. Basically the package we are using hands us the rendered TeX expression as a raster image, with no information about what part of the expression is where. Although I don't know how to solve this problem, contributions that address it are welcome.

As for a workaround, maybe you can render the equation with and without the variable of interest and use the differences of overall bounding boxes.
